### PR TITLE
Use newest buildpack in Makefiles and newest images in Dockerfiles of Application Connectivity

### PR DESCRIPTION
**Pull request by franpog859**
**Description**

Changes proposed in this pull request:

- Use newest buildpack in Makefiles of Application Connectivity
- Use newest images in Dockerfiles of Application Connectivity
- Add `licenses` to .gitignores 
- Adjust old development scripts to match new Makefile approach

**Related issue(s)**

See the issue #6113

### CODE REVIEW

File: components/application-registry/scripts/can-i-commit.sh [hunk 1]
```diff
@@ -24,15 +24,15 @@ if [ $? != 0 ]
 fi
 
 ##
-# MAKE BUILD
+# MAKE VERIFY
```
Comment (Szymongib) [inline components/application-registry/scripts/can-i-commit.sh]: Do we still need this file? I guess we can do basically the same just by running `make verify`.

File: components/connector-service/scripts/can-i-commit.sh [hunk 2]
```diff
@@ -24,15 +24,15 @@ if [ $? != 0 ]
 fi
 
 ##
-# MAKE BUILD
+# MAKE VERIFY
```
Comment (Szymongib) [inline components/connector-service/scripts/can-i-commit.sh]: Same as in `application-registry`, not sure if we still need it.

File: tests/application-connector-tests/Dockerfile [hunk 3]
```diff
@@ -1,13 +1,14 @@
-FROM golang:1.10-alpine as builder
+FROM golang:1.13-alpine as builder
 
 ENV SRC_DIR=/go/src/github.com/kyma-project/kyma/tests/application-connector-tests
+ENV CGO_ENABLED=0
```
Comment (Szymongib) [inline tests/application-connector-tests/Dockerfile]: I think we can do it the same as in other components, setting this var only for building binary like this: `RUN CGO_ENABLED=0 go test...`

File: components/application-registry/scripts/can-i-commit.sh (hunk 1 above)
Comment (franpog859) [reply components/application-registry/scripts/can-i-commit.sh]: Done!

File: components/connector-service/scripts/can-i-commit.sh (hunk 2 above)
Comment (franpog859) [reply components/connector-service/scripts/can-i-commit.sh]: Done!

File: tests/application-connector-tests/Dockerfile (hunk 3 above)
Comment (franpog859) [reply tests/application-connector-tests/Dockerfile]: Done!

**Pull request merged**